docs/source/configuration/telemetry/exporters/metrics/overview.mdx

```diff
@@ -123,6 +123,12 @@ You can add custom attributes (OpenTelemetry) and labels (Prometheus) to the `ap
 * a value from a context
 * a value from the request or response body ([JSON path](https://goessner.net/articles/JsonPath/))
 
+<Note>
+
+Use [resource attributes](#resource) instead to provide information about telemetry resources.
```

Comment:
Maybe expannd on the sorts of things that should be used in resources, such as env or host.